.rubocop.yml

```diff
@@ -7,17 +7,23 @@ AllCops:
   DisplayCopNames: true
 
 Layout/LineLength:
-  Max: 120
+  Max: 500
 Metrics/MethodLength:
-  Max: 20
-Metrics/AbcSize:
   Max: 50
+Metrics/AbcSize:
+  Max: 200
 Metrics/ClassLength:
-  Max: 150
+  Max: 1000
```

Comment:
You should not override the default configurations. So instead of it, do inline disabling. Which will urge you to resolve the error later.